project/Dependencies.scala

```diff
@@ -37,6 +39,8 @@ object Dependencies {
 
   val akkaDiagnostics = Seq(
     commonsLang, // for levenshtein distance impl
+    akkaRemoting,
+    akkaClusterMetrics,
```

Comment:
those two dependencies should be `% Provided`

Reply:
changed